### ISSUE TITLE
BELL - Institution Acronym

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/new-collections#281]
Institution to register at [GRSciColl](https://www.gbif.org/grscicoll/institution/search)

**Comment by Jegelewicz** [comment]
GRSciColl no longer active. What are they using now and is it what they want to continue using?

**Comment by Jegelewicz** [comment]
Found it! https://www.gbif.org/grscicoll/institution/e46e34a4-32b8-4a0c-89a7-06f14fcde702

Acronym is MMNH Should be OK in Arctos if that's what they want to keep.

**Comment by ccicero** [comment]
Yes, that's the acronym they use in VertNet. I can confirm they want to keep it. I don't see their Prospective Collection Form in the list though?
